files/jsondata/L10n-Common.json

```
@@ -170,6 +186,7 @@
     "es": "Nuestros voluntarios aún no han traducido este artículo al Español. ¡Únete a nosotros y ayuda a traducirlo!",
     "fr": "Nos volontaires n'ont pas encore traduit cet article en français. Rejoignez-nous et aidez-nous à le faire&nbsp;!",
     "ja": "まだボランティアによって日本語に翻訳されていない記事です。 MDN に参加して、翻訳してみませんか？",
+    "ko": "이 문서는 아직 한국어로 번역되지 않았습니다. 저희와 함께 끝낼 수 있도록 도와주세요!",
```

```suggestion
    "ko": "이 문서는 아직 한국어로 번역되지 않았습니다. 저희와 함께 번역을 끝낼 수 있게 도와주세요!",
```

조금 더 자연스럽게 "번역을"을 추가하면 좋을 것 같아요!